stable/awsebscsiprovisioner/templates/storageclass.yaml

```diff
@@ -21,3 +21,7 @@ parameters:
   {{- if .Values.storageclass.encrypted }}
   encrypted: {{ .Values.storageclass.encrypted | quote }}
   {{- end }}
+{{- with .Values.storageclass.allowedTopologies }}
+allowedTopologies:
```

Comment:
Let's allow a user to not set any allowed topologies at all. This would be the default case for on-prem. I.e., surround this with an if-cluase
```
{-if .Values.storageclass.allowedTopologies }}
allowedTopologies:
  [...]
{{- end }}
```

Reply:
It'll have the same behavior but I can still add it :)

Reply:
Done